src/app/app.component.ts

```diff
@@ -26,13 +26,16 @@ export class AppComponent implements OnInit  {
     this.languageService.setDefaultLanguage();
     this.navigateToStartingPositionOnPage();
     this.titleAndMetaTagsService.useTitleMetasData();
-    this.searchSearch.searchSubject.subscribe(this.openSearchSubscription.bind(this));
+    this.searchSearch.searchSubject.subscribe(this.openSearchSubscription);
     this.uiActionsService.stopScrollingSubject.subscribe(data => this.toggle = data);
   }
 
   private openSearchSubscription(isSearchExpanded: boolean): void {
-    isSearchExpanded ? (this.elRef.nativeElement.ownerDocument.body.style.overflow = 'hidden') :
-             (this.elRef.nativeElement.ownerDocument.body.style.overflow = 'auto');
+    if (isSearchExpanded) {
```

Comment:
please revert changes lines 29 - 34 to ternary operator, but update it to correct way.

Reply:
done